README.md

```diff
@@ -1 +1,11 @@
-``` shell
g++ -static lib_sample_use.cpp -std=c++11 -lfastFM -lglog -pthread -I. -L.
```
\ No newline at end of file
+Build:
+
+    > $GCC_MUSL_ROOT/x86_64-linux-musl-g++ -static -std=c++11 lib_sample_use.cpp libfastFM.a libglog.a -o foo
```

Comment:
> Since static build doesn't seem to work my only suggestion is to try shared library. 

fastFM build with musl as shared library? All fine with me as long as we can ship all dependencies.
I mean if cloning this repo & a regular g++ install is sufficient then we should be fine.